apps/demo/netlify.toml

```diff
@@ -34,3 +34,10 @@ ignore = """\
     ./packages/web/ \
     ./.gihub/workflows/ \
     """
+
+[[headers]]
+# you can configure the correct paths here
+for = "/*"
+[headers.values]
+# see a list of allowed values here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin
+access-control-allow-origin = "*"
```

Comment:
**suggestion (if-minor):** This is ok for testing. After we check that this is working, can we please add the actual domain?

Reply:
outdated